irods_builder.el8.Dockerfile

```diff
@@ -0,0 +1,64 @@
+# syntax=docker/dockerfile:1.5
+
+FROM rockylinux:8
+
+SHELL [ "/usr/bin/bash", "-c" ]
+
+# Make sure we're starting with an up-to-date image
+RUN --mount=type=cache,target=/var/cache/dnf,sharing=locked \
+    --mount=type=cache,target=/var/cache/yum,sharing=locked \
+    dnf update -y || [ "$?" -eq 100 ] && \
+    rm -rf /tmp/*
+
+RUN --mount=type=cache,target=/var/cache/dnf,sharing=locked \
+    --mount=type=cache,target=/var/cache/yum,sharing=locked \
+    dnf install -y \
+        epel-release \
+    && \
+    rm -rf /tmp/*
+
+RUN --mount=type=cache,target=/var/cache/dnf,sharing=locked \
+    --mount=type=cache,target=/var/cache/yum,sharing=locked \
+    dnf install -y \
+        redhat-lsb-core \
+        wget \
+        rpm-build \
+    && \
+    rm -rf /tmp/*
+
+RUN --mount=type=cache,target=/var/cache/dnf,sharing=locked \
+    --mount=type=cache,target=/var/cache/yum,sharing=locked \
+    dnf install -y \
+        dnf-plugin-config-manager \
+        dnf-plugins-core \
+    && \
+    rpm --import https://packages.irods.org/irods-signing-key.asc && \
+    dnf config-manager -y --add-repo https://packages.irods.org/renci-irods.yum.repo && \
+    dnf config-manager -y --set-enabled renci-irods && \
+    dnf config-manager -y --set-enabled powertools && \
+    rm -rf /tmp/*
+
+ARG irods_version=4.3.2-0.el8
```

Comment:
Need words about these options. Or we just open an issue and address it later.

Reply:
done in https://github.com/irods/irods_client_http_api/issues/408